mesonbuild/backend/ninjabackend.py

```diff
@@ -1979,6 +1979,15 @@ def generate_rust_target(self, target: build.BuildTarget) -> None:
         else:
             verbatim = ''
 
+        def _link_library(libname: str, static: bool, modifiers: T.Optional[T.List[str]] = None, bundle: bool = False):
+            _type = 'static' if static else 'dylib'
```

Comment:
nitpick: usually, we use _trailing_ underscore in Python to distinguish identifiers from reserved keywords.

https://peps.python.org/pep-0008/#descriptive-naming-styles

Reply:
done